gulpfile.js

```diff
@@ -246,6 +259,13 @@ var buildDoc = function () {
     });
 };
 
+function transformRelativeLinks(buffer, ext) {
+    const RELATIVE_LINK_REPLACE_REGEXP = /\(\.\/(.+)\..+\)/gm;
```

Comment:
I wonder if we should tweak this slightly to `/\(\.\/([^)]+)\.[^)]+\)/gm` so that we can handle two instances of links in a line .. in case it comes up.

`[Foo](./foo.md) [Bar](./bar.md)`

Reply:
Yeah, that makes sense to me, this is some tier 1 regex.